self_play.py

```diff
@@ -175,19 +176,26 @@ def mcts_decision(policy, board, mcts_simulations, mcts_tree, temperature, model
             ps.append(p)
         selected_a = np.random.choice(moves, size=1, p=ps)[0]
     elif temperature == 0:
-        _, _, selected_a = max((dic['count'], dic['mean_value'], a) for a, dic in mcts_tree['subtree'].items())
+        if skip_resign:
+            _, _, selected_a = max((dic['count'], dic['mean_value'], a) 
+                    for a, dic in mcts_tree['subtree'].items()
+                    if a<SIZE*SIZE)
+        elif value<0.05:
+            selected_a=SIZE*SIZE
```

Comment:
The 0.05 should be in the configuration.

I have a feeling you're confusing resigning (declaring you have lost) with passing (which is a valid move and can be a winning move)